### PR TITLE
Document Prometheus metrics and OLAP Flow events

### PR DESCRIPTION
**Pull request by hasit**
### Description of change

Closes: #720 

##### Checklist

- [ ] Tested in playground or other setup
- [x] Documentation is changed or added
- [ ] Tests and/or benchmarks are included
- [ ] Breaking changes

- - -
This change is [Reviewable](https://reviewable.io/reviews/fluxninja/aperture/878)

### CODE REVIEW

File: docs/content/references/metrics/agent.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+| policy_hash      | 5kZjjSgDAtGWmLnDT67SmQhZdHVmz0+GvKcOGTfWMVo= | Hash of the policy used for checking integrity of the policy.    |
+| classifier_index | 0, 1                                         | Index of the classifier in order of specification in the policy. |
+
+## Dataplane Summary
+
+### FlowControl Metrics
+
+| Name                             | Type    | Labels        | Unit            | Description                                     |
+| -------------------------------- | ------- | ------------- | --------------- | ----------------------------------------------- |
+| flowcontrol_requests_total       | Counter |               | count (no unit) | Total number of aperture check requests handled |
```
Comment (tanveergill) [inline docs/content/references/metrics/agent.md:84]: @hasit,
All of the labels are not applicable to each metric. Please refer the labels for each metric carefully: https://github.com/fluxninja/aperture/blob/72ad678cc64ddedfd53498b0a4eef0b6cb65bf24/pkg/policies/flowcontrol/api/base/metrics.go#L55

Please do this for all the other metrics as well

File: docs/content/references/metrics/agent.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+| distcache_entries_total | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Total number of entries in the DMap                                                   |
+| distcache_delete_hits   | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Number of deletion requests resulting in an item being removed in the DMap            |
+| distcache_delete_misses | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Number of deletion requests for missing keys in the DMap                              |
+| distcache_get_misses    | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Number of entries that have been requested and not found in the DMap                  |
+| distcache_get_hits      | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Number of entries that have been requested and found present in the DMap              |
+| distcache_evicted_total | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Total number of entries removed from cache to free memory for new entries in the DMap |
```
Comment (tanveergill) [inline docs/content/references/metrics/agent.md:106]: Labels such as instance are present on all Agent metrics. Might be a good idea to refer to metrics explorer in Prometheus

File: docs/content/references/metrics/agent.md (hunk 1 above)
Comment (hasit) [reply docs/content/references/metrics/agent.md]: Done. Confirmed and updated the tables.

File: docs/content/references/metrics/agent.md (hunk 2 above)
Comment (hasit) [reply docs/content/references/metrics/agent.md]: Done.

**Review by harjotgill — APPROVED** [review]

**Review by tanveergill — APPROVED** [review]

**Pull request merged**